packages/narwallets/src/lib/narwallets.ts

```diff
@@ -0,0 +1,281 @@
+import { isMobile } from "is-mobile";
+import {
+  WalletModuleFactory,
+  WalletBehaviourFactory,
+  InjectedWallet,
+  Account,
+  Action,
+  FinalExecutionOutcome,
+} from "@near-wallet-selector/core";
+import { providers } from "near-api-js";
```

Comment:
I don't think this is being used anywhere, we can remove it otherwise it will throw linting errors.

There are more linting issues in this file, to find them try `yarn nx run narwallets:lint` .

After you remove the above import try `yarn lint:fix` not sure if it will automatically fix them for you but it could work.

Reply:
Just solved the conflicts and solved the linting issues with `yarn` lint:fix`